api/collections/serializers.py

```diff
@@ -143,7 +143,7 @@ def make_instance_obj(self, obj):
         return {'data': [
             pointer for pointer in
             obj.nodes_pointer
-            if not pointer.node.is_deleted and not pointer.node.is_folder
+            if not pointer.node.is_deleted and not pointer.node.is_collection
```

Comment:
This is cleanup.  The dashboard branch that I am merging into has failing tests - that had nothing to do with my changes.  Collections are the new name for folders.